### DISCUSSION TITLE
How to return custom data in query

### DISCUSSION THREAD

**Discussion by bionyxxx** [rappasoft/laravel-livewire-tables#446] [Q&A] [answered]
Hello,

I want to ask how to return data on query but not from model/database, for example I retrieve data via API

thx

**Comment by rappasoft** [comment, marked as answer]
Currently the package only supports Eloquent.

**Reply by ArtMin96** [reply]
Hi @rappasoft . Any news related to this discussion?

**Reply by tonypartridge** [reply]
You would be best fetching all the data and putting it into the database.

**Reply by ArtMin96** [reply]
@tonypartridge In Stripe has thousands of payments, plus Stripe has a convenient dashboard. I think after that it makes no sense to send a webhook for every payment and save it in the database and then show it.

**Reply by tonypartridge** [reply]
@ArtMin96  checkout [Squisjh](https://github.com/calebporzio/sushi)
